lib/src/go_true_exception.dart

```diff
@@ -1,18 +1,18 @@
-class GotrueError {
+class GoTrueException {
   final String message;
   final String? statusCode;
 
-  GotrueError(this.message, {this.statusCode});
+  GoTrueException(this.message, {this.statusCode});
```

Comment:
This can be const. Like:
```suggestion
  const GoTrueException(this.message, {this.statusCode});
```

> **Note**: this will trigger other warnings about const constructors